CHANGELOG.md

```diff
@@ -12,18 +12,18 @@ modified Semantic Versioning scheme. See the "Versioning scheme" section of the
 ### Added
 
 ### Changed
-Added styles to address IE11 issues for height on single-block layout section columns, logo width on wide logos in the header, and the search button overlapping the search input
 
 ### Deprecated
 
 ### Removed
 
 ### Fixed
-- RIGA-52: Removed backticks from html.html.pl.twig to avoid ie11 uncompiled, non-polyfilled JS errors.
-- RIGA-52: Move palette class to HTML rather than body.
 
 ### Security
 
+## [0.5.5] - 2021-04-26
+- RIGA-60: Added styles to address IE11 issues for height on single-block layout section columns, logo width on wide logos in the header, and the search button overlapping the search input
```

Comment:
Just looking for the "Changed" heading here

Reply:
🤦